03b_model/src/run_model.py

```diff
@@ -194,13 +369,59 @@ def train_model(prepped_model_io_data_file, inputs, seq_len,
                 out_dir, run_id,                       
                 train_start_date, train_end_date,
                 val_start_date, val_end_date,
-                test_start_date, test_end_date):
-    
+                test_start_date, test_end_date, inc_ante):
+    '''
+    write modeling parameters to a .txt file within out_dir/run_id, train the model,
+    save the weights, and save a plot of the losses
```

Comment:
It may be better to split into 3 functions: write parameters, train model and save weights, plot losses. I'm mostly thinking a separate plot function can help if you want to change aesthetics of the plot and not have to retrain.